<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/prefijoNuevo/images/main_menu_image.png);</string>
  </property>
  <widget class="QPushButton" name="buttonGame1main">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>900</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 125, 17);</string>
   </property>
   <property name="text">
    <string>JUEGO 1 (1)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExitApp">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>900</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(92, 53, 102);</string>
   </property>
   <property name="text">
    <string>SALIR (X)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGame2main">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>900</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 121, 0);</string>
   </property>
   <property name="text">
    <string>JUEGO 2 (2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/prefijoNuevo/main_menu_image.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>buttonGame1main</zorder>
  <zorder>buttonExitApp</zorder>
  <zorder>buttonGame2main</zorder>
 </widget>
 <resources>
  <include location="assets/source.qrc"/>
 </resources>
 <connections/>
</ui>
